### PR TITLE
support quic and latency in hubtest

### PR DESCRIPTION
**Pull request by edisonxiang**
**What type of PR is this?**
/kind test

**What this PR does / why we need it**:
* support quic in hubtest
   quic is passing the data by bytes[] and websocket is passing the data by json
* support latency calculation in hubtest
   support 50%, 90%, 99% and 100% throughputs latency
* remove the Resource Limits for EdgeController
   avoid the perf test bottleneck

### CODE REVIEW

File: tests/e2e/utils/common.go [hunk 1]
```diff
@@ -144,10 +144,10 @@ func cloudcoreDeploymentSpec(imgURL, configmap string, replicas int) *apps.Deplo
 								v1.ResourceName(v1.ResourceCPU):    resource.MustParse("100m"),
 								v1.ResourceName(v1.ResourceMemory): resource.MustParse("512Mi"),
 							},
-							Limits: v1.ResourceList{
+							/*Limits: v1.ResourceList{
```
Comment (sids-b) [inline tests/e2e/utils/common.go]: This is a good change. EdgeController is a cloud component and may not resource constraint environment

**Comment by sids-b** [comment]
ping @m1093782566 @pavan187 for review

File: tests/e2e/utils/common.go (hunk 1 above)
Comment (m1093782566) [reply tests/e2e/utils/common.go]: If we don't need these test scenarios, why not delete them?

Comment (edisonxiang) [reply tests/e2e/utils/common.go]: OK. will update later.

**Pull request merged**